packages/client/test/net/peerpool.spec.ts

```diff
@@ -65,24 +71,24 @@ describe('[PeerPool]', async () => {
   })
 
   it('should check contains', () => {
-    const peer = new Peer('abc')
+    const peer: any = new Peer('abc')
```

Comment:
So I assume that these (and some other) any castings are necessary to get things working?

Please really double-check if we really need to have, adding `any` is always not so nice (but if needed, so be it).

Reply:
The test will pass without the `as any`.
We get a ts error flag here, but it doesn't actually stop the test from running.